### ISSUE TITLE
Allow comments to be deleted during a revision

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Revisions#618]
This could perhaps be a sidebar button or link?

> In regards to comments, it would be helpful if comments could be deleted when a revision is published. Sometimes the updates or rewrite makes the comments outdated or irrelevant.

**Comment by agapetry** [comment]
@andergmartins Is comment deletion still something you're interested in supporting within PublishPress?

**Comment by stevejburge** [comment]
@agapetry I think this is worth doing, yes. Ola is charge of PublishPress now

**Comment by andergmartins** [comment]
@agapetry Yes, we added it as a feature https://github.com/publishpress/PublishPress/issues/277, we should keep supporting that
